From a68da37a138c6dfa4af73bec94a628988f79f3a0 Mon Sep 17 00:00:00 2001
From: Juan A. Suarez Romero <jasuarez@igalia.com>
Date: Wed, 07 Sep 2011 18:08:22 +0000
Subject: apple-trailers: set mime-type

Mime-type is set to "video/mp4".

From the webpage, seems the .mov files have this mimetype.

And .m4v has usually this mimetype too.

Signed-off-by: Juan A. Suarez Romero <jasuarez@igalia.com>
---
diff --git a/src/media/apple-trailers/grl-apple-trailers.c b/src/media/apple-trailers/grl-apple-trailers.c
index 949f188..9977094 100644
--- a/src/media/apple-trailers/grl-apple-trailers.c
+++ b/src/media/apple-trailers/grl-apple-trailers.c
@@ -361,6 +361,8 @@ build_media_from_movie (xmlNodePtr node, gboolean xlarge)
   grl_media_set_certificate (media, movie_rating);
   grl_media_set_studio (media, movie_studio);
 
+  grl_media_set_mime (media, "video/mp4");
+
   /* FIXME: Translation */
   grl_media_set_license (media, movie_copyright);
 
--
cgit v0.9.0.2
